uiplib/GUI.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, settings):
         self.settings = settings
         # base window
         self.root = Tk()
-        self.root.resizable(width=False, height=False)
+        self.root.resizable(width=True, height=True)
```

Comment:
hey so the issue was about responsiveness as stated here: `Improve it to have resizability while not sacrificing frame bounds and provide responsiveness.` not just about changing the resizable attribute